src/helm/benchmark/presentation/run_specs_dec2023.conf

```diff
@@ -0,0 +1,169 @@
+  {description: "math:model=text_code,subject=intermediate_algebra,level=4,use_chain_of_thought=True", priority: 4}
+  {description: "math:model=text_code,subject=algebra,level=4,use_chain_of_thought=True", priority: 4}
+  {description: "math:model=text_code,subject=prealgebra,level=4,use_chain_of_thought=True", priority: 4}
+  {description: "math:model=text_code,subject=geometry,level=4,use_chain_of_thought=True", priority: 4}
+  {description: "math:model=text_code,subject=counting_and_probability,level=4,use_chain_of_thought=True", priority: 4}
+  {description: "math:model=text_code,subject=precalculus,level=4,use_chain_of_thought=True", priority: 4}
+
+  {description: "math:model=text_code,subject=number_theory,level=5,use_chain_of_thought=True", priority: 3}
+  {description: "math:model=text_code,subject=intermediate_algebra,level=5,use_chain_of_thought=True", priority: 3}
+  {description: "math:model=text_code,subject=algebra,level=5,use_chain_of_thought=True", priority: 3}
+  {description: "math:model=text_code,subject=prealgebra,level=5,use_chain_of_thought=True", priority: 3}
+  {description: "math:model=text_code,subject=geometry,level=5,use_chain_of_thought=True", priority: 3}
+  {description: "math:model=text_code,subject=counting_and_probability,level=5,use_chain_of_thought=True", priority: 3}
+  {description: "math:model=text_code,subject=precalculus,level=5,use_chain_of_thought=True", priority: 3}
+
+  # GSM
+  {description: "gsm:model=text_code", priority: 2}
+
+  # HumanEval
+  {description: "code:model=text_code,dataset=humaneval", priority: 1}
+
+  # LSAT
+  {description: "lsat_qa:model=text_code,task=all", priority: 2}
+  {description: "lsat_qa:model=text_code,task=grouping", priority: 3}
+  {description: "lsat_qa:model=text_code,task=ordering", priority: 3}
+  {description: "lsat_qa:model=text_code,task=assignment", priority: 3}
+  {description: "lsat_qa:model=text_code,task=miscellaneous", priority: 3}
+
+  # LegalSupport
+  {description: "legal_support:model=text_code", priority: 2}
+
+  # MedQA
+  {description: "med_qa:model=text_code", priority: 2}
+
+  # WMT
+  {description: "wmt_14:language_pair=en-cs,model=text", priority: 3}
+  {description: "wmt_14:language_pair=en-de,model=text", priority: 3}
+  {description: "wmt_14:language_pair=en-fr,model=text", priority: 3}
+  {description: "wmt_14:language_pair=en-hi,model=text", priority: 3}
+  {description: "wmt_14:language_pair=en-ru,model=text", priority: 3}
+  {description: "wmt_14:language_pair=cs-en,model=text", priority: 2}
+  {description: "wmt_14:language_pair=de-en,model=text", priority: 2}
+  {description: "wmt_14:language_pair=fr-en,model=text", priority: 2}
+  {description: "wmt_14:language_pair=hi-en,model=text", priority: 2}
+  {description: "wmt_14:language_pair=ru-en,model=text", priority: 2}
+
+  # RealToxicityPrompts
+  {description: "real_toxicity_prompts:model=text", priority: 2}
+
+  # BBQ
+  {description: "bbq:model=text,subject=all", priority: 2}
```

Comment:
Could keep but need to be careful: what one cares about here is not just the accuracy on BBQ, but the specific bias measures. So need to make sure the schema includes those (and, again, this is complicated because the semantics are no longer just accuracy)